streams/src/test/java/org/apache/kafka/streams/StreamsConfigTest.java

```diff
@@ -81,7 +81,6 @@ public void setUp() {
         props.put(StreamsConfig.DEFAULT_VALUE_SERDE_CLASS_CONFIG, Serdes.String().getClass().getName());
         props.put("key.deserializer.encoding", "UTF8");
         props.put("value.deserializer.encoding", "UTF-16");
-        streamsConfig = new StreamsConfig(props);
```

Comment:
Was this really necessary?

Reply:
As you can see below, `new StreamsConfig(props);` is called in every test method of this suite. So, this line is redundant.

More importantly, it generates duplicated log messages and makes log message related test cases hard to validate.